frontend/src/components/PaymentForm.tsx

```diff
@@ -68,21 +68,17 @@ class PaymentForm extends Component {
 
     console.log("Obtained key. Now encrypting credentials");
     const decodedPublicKey = atob(pciEncryptionKey.publicKey);
-    const publicKey = await openpgp.readKey({ armoredKey: decodedPublicKey });
-
-    return openpgp
-      .encrypt({
-        message: await openpgp.createMessage({
-          text: JSON.stringify(cardCredentials),
-        }),
-        encryptionKeys: publicKey,
-      })
-      .then((ciphertext) => {
-        return {
-          encryptedCredentials: btoa(ciphertext),
-          keyId: pciEncryptionKey.keyId,
-        };
-      });
+    const options = {
```

Comment:
FYI -- this is some cleanup work I had to do with openpgp. I was likely using the wrong version/package before